modulemd/v2/include/modulemd-2.0/modulemd-module.h

```diff
@@ -33,6 +33,18 @@ G_DECLARE_FINAL_TYPE (
   ModulemdModule, modulemd_module, MODULEMD, MODULE, GObject)
 
 
+/**
+ * modulemd_module_index_get_stream_names_as_strv: (rename-to modulemd_module_index_get_stream_names)
+ * @self: This #ModulemdModule
+ *
+ * Returns: (transfer full): An ordered list of string keys in this module.
```

Comment:
```suggestion
 * Returns: (transfer full): An ordered list of stream names in this module.
```

Reply:
I'm guessing you copied this from `modulemd_module_index_get_module_names_as_strv()`, but that was a bad description there too. I'll fix that later.